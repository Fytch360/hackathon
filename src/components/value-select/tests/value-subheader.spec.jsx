import React from 'react'
import { shallow } from 'enzyme'

import { ValueSubheader } from '..'

describe('<ValueSubheader />', () => {

    test('Should be available', () => {
        expect(ValueSubheader).toBeDefined()
    })

    test('Snapshot', () => {
        const wrapper = shallow(<ValueSubheader title="mmm-title-example" size="lg" />)

        expect(wrapper).toMatchSnapshot()
    })
})
