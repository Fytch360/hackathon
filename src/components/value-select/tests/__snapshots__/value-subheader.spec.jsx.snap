// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ValueSubheader /> Snapshot 1`] = `
<GroupTitleStyled
  size="md"
>
  mmm-title-example
</GroupTitleStyled>
`;
